Jantimizer/QueryOptimiserTest/JoinCostTest.cs

```diff
@@ -0,0 +1,258 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using QueryOptimiser;
+using Histograms;
+using Histograms.Managers;
+using QueryParser;
+using QueryParser.QueryParsers;
+using QueryParser.Models;
+using QueryOptimiserTest;
+
+namespace CostTest;
+
+[TestClass]
+public class JoinCostTest
```

Comment:
Again, this should perhaps be specified as being the Join Cost for Postgres Equi-Depth histograms and queries.